src/views/sell/helpers/handleMultipleOffersPublished.ts

```diff
@@ -10,9 +10,11 @@ export const handleMultipleOffersPublished = async (result: SellOffer[], offerDr
   useOfferPreferences.getState().setMulti(undefined)
   result.forEach((offer) => saveOffer({ ...offerDraft, ...offer }))
 
-  const internalAddress = await peachWallet.getNewInternalAddress()
+  const internalAddress = await peachWallet.getInternalAddress()
+  const newInternalAddress = await peachWallet.getInternalAddress(internalAddress.index + 10)
```

Comment:
Why are we doing this?

Reply:
To avoid having the next change address registered to funding multiple escrows.

Use case is, if a user makes a regular transaction with a change output inbetwee, they would accidentally "fund multiple escrows". I first considered a much higher index but that would cause the last unused address index to grow too quickly.

With bdk version 1.0.0 we will change this behaviour to specifically watch an address at a specific index